doc/usage.rst

```diff
@@ -37,6 +37,18 @@ After you do this, you name the spec file to |PyInstaller| instead of the script
 
     ``pyinstaller myscript.spec``
 
+The :file:`myscript.spec` file contains most of the information provided by the options that were specified
+when ``pyinstaller`` (or ``pyi-makespec``) was run with the script file as the argument.
+You typically do not need to specify any options when running ``pyinstaller`` with the spec file.
+Only the following command-line options (described below) have an effect when building from a spec file:
```

Comment:
To avoid duplication (and errors when something changes), I suggest not listing the options here, but refer to :ref:`Using Spec Files` again. Something like
```
Only :ref:`a few command-line options <Using Spec Files>` have an effect when building from a spec file.
```